asset.go

```diff
@@ -0,0 +1,156 @@
+package uipath
+
+import (
+	"encoding/json"
+	"fmt"
+	"strconv"
+)
+
+const (
+	ValueTypeText       = "Text"
+	ValueTypeInteger    = "Integer"
+	ValueTypeBool       = "Bool"
+	ValueTypeCredential = "Credentials"
+
+	ValueScopeGlobal = "Global"
+)
+
+// AssetHandler struct defines what the asset handler looks like
+type AssetHandler struct {
+	Client   *Client
+	FolderId uint
+}
+
+// Asset struct defines what the asset model looks like
+type Asset struct {
+	ID                 uint     `json:"Id"`
+	Name               string   `json:"Name"`
+	CanBeDeleted       bool     `json:"CanBeDeleted,omitempty"`
+	ValueScope         string   `json:"ValueScope,omitempty"`
+	ValueType          string   `json:"ValueType"`
+	Value              string   `json:"Value,omitempty"`
+	StringValue        string   `json:"StringValue,omitempty"`
+	BoolValue          bool     `json:"BoolValue,omitempty"`
+	IntValue           int      `json:"IntValue,omitempty"`
+	CredentialUsername string   `json:"CredentialUsername,omitempty"`
+	CredentialPassword string   `json:"CredentialPassword,omitempty"`
+	ExternalName       string   `json:"ExternalName,omitempty"`
+	CredentialStoreId  int      `json:"CredentialStoreId,omitempty"`
+	HasDefaultValue    bool     `json:"HasDefault,omitempty"`
+	Description        string   `json:"Description,omitempty"`
+	FolderCount        int      `json:"FolderCount,omitempty"`
+	Tags               []string `json:"Tag,omitempty"`
+	KeyValueList       []string `json:"KeyValueList,omitempty"`
+}
+
+// AssetList struct defines what the asset list looks like
+type AssetList struct {
+	Count int     `json:"@odata.count"`
+	Value []Asset `json:"Value"`
+}
+
+// Tag struct defines what the tag item looks like
+type Tag struct {
+	Name         string `json:"Name"`
+	DisplayName  string `json:"DisplayName"`
+	Value        string `json:"Value"`
+	DisplayValue string `json:"DisplayValue,omitempty"`
+}
+
+// GetByID fetches the asset by id
+func (a *AssetHandler) GetByID(ID uint) (Asset, error) {
+	var asset Asset
+
+	url := fmt.Sprintf("%s%s(%d)", a.Client.URLEndpoint, "Assets", ID)
+
+	resp, err := a.Client.SendWithAuthorization("GET", url, nil, a.buildHeaders(), map[string]string{})
+	if err != nil {
+		return asset, err
+	}
+
+	if err = json.Unmarshal(resp, &asset); err != nil {
+		return asset, err
+	}
+
+	return asset, nil
```

Comment:
We can simplify this error handling here.

```suggestion
	err = json.Unmarshal(resp, &asset)

	return asset, err
```